src/types/index.ts

```diff
@@ -161,6 +161,7 @@ export type CreatePaymentIntentParams = CreatePaymentIntentIOSParams & {
   transferGroup?: string;
   metadata?: Record<string, string>;
   paymentMethodOptions?: PaymentMethodOptions;
+  captureMethod?: string;
```

Comment:
Done https://github.com/stripe/stripe-terminal-react-native/pull/443/commits/d760bf123376b5a31058b8dcd092227c087fd2f5#diff-a9ba9cbedd19c9f66d564d2e89912890209b98f0a7ef19187877d2587300e476

Reply:
As a side note, maybe it would be nicer if you update in a future version the type for [paymentMethodTypes](https://github.com/stripe/stripe-terminal-react-native/blob/main/src/types/index.ts#L167) as well. I see you only allow 
`'cardPresent' | 'interacPresent' | 'card'` for `paymentMethodType` but on the backend the list is bigger https://stripe.com/docs/api/payment_intents/create#create_payment_intent-payment_method_types

I'm also confused why it's camel case for the SDK, I tested with snake case and it worked fine!

Reply:
> As a side note, maybe it would be nicer if you update in a future version the type for [paymentMethodTypes](https://github.com/stripe/stripe-terminal-react-native/blob/main/src/types/index.ts#L167) as well. I see you only allow `'cardPresent' | 'interacPresent' | 'card'` for `paymentMethodType` but on the backend the list is bigger https://stripe.com/docs/api/payment_intents/create#create_payment_intent-payment_method_types

Terminal really only supports `card_present` and `interac_present` (we may even want to remove "card"). Terminal is meant to support in-person (aka, card-present) payments. Type `card` is used for card-not-present payments (e.g., for e-commerce).

> I'm also confused why it's camel case for the SDK, I tested with snake case and it worked fine!

Yeah, that's a good point. It looks like for Stripe API responses, we translate to the camel-case versions [0] (only for Setup Attempts and Refunds -- not Payment Intents), but when setting the params, we expect and use snake case [1]. They are being passed through directly to the native iOS SDK, and they should be snake case (since that's also what the native SDK and backend expect).

Agree this could use some cleaning up. Thanks for pointing it out.

[0] https://github.com/stripe/stripe-terminal-react-native/blob/0bca5f0db2a62f7bd9b06d8f3abfec887b0a23f3/ios/Mappers.swift#L377-L384
[1] https://github.com/stripe/stripe-terminal-react-native/blob/29bcc5701fad684f4beb46408d0a326657733481/dev-app/src/screens/CollectCardPaymentScreen.tsx#L113-L116